src/Illuminate/Pagination/LengthAwarePaginator.php

```diff
@@ -128,7 +128,7 @@ public function render(Presenter $presenter = null)
 
         $presenter = $presenter ?: new BootstrapThreePresenter($this);
 
-        return $presenter->render();
+        return new Expression($presenter->render());
```

Comment:
can't we just do `if (class_exists(\Illuminate\View\Expression::class) {` cause people using it outside of Laravel most likely doesn't use blade at the same time.